tests/Feature/Proxy/ShipmentProxyTest.php

```diff
@@ -368,6 +368,26 @@ public function testClientCalls()
         $this->assertEquals(2, $this->clientCalls['https://api/v1/shipments/shipment-id-1']);
     }
 
+    /** @test */
+    public function testRegisterAt()
+    {
+        $this->assertEquals(
+            1337,
+            $this->shipmentProxy->setRegisterAt(1337)->getRegisterAt()->getTimestamp()
+        );
+        $this->assertEquals(
+            (new \DateTime('+1 year'))->getTimestamp(),
+            $this->shipmentProxy->setRegisterAt('+1 year')->getRegisterAt()->getTimestamp()
```

Comment:
I think there is a possibility here that timestamps do not match, if execution takes a second.